src/applications/facility-locator/components/search-results/LocationPhoneLink.jsx

```diff
@@ -36,6 +34,25 @@ const renderPhoneNumber = (
         </a>
       ) : null}
     </div>
+  ) : (
+    <dfn>
+      {from === 'FacilityDetail' && <i className={`fa fa-${icon}`} />}
+      {title && <strong>{title}: </strong>}
+      {!isCCProvider && phone.replace(re, '$1-$2-$3 $4$5').replace(/x$/, '')}
+      <br />
+      {from === 'FacilityDetail' && <i className="fa fa-fw" />}
+      {subTitle}
+      {isCCProvider &&
+        ` ${phone.replace(re, '$1-$2-$3 $4$5').replace(/x$/, '')}`}
+      {from === 'FacilityDetail' ? (
+        <a
+          href={`tel:${phone.replace(/[ ]?x/, '')}`}
+          className={altPhone && 'facility-phone-alt'}
+        >
+          {phone.replace(re, '$1-$2-$3 $4$5').replace(/x$/, '')}
```

Comment:
Since the regex is more than once, can we make this a helper function? 

```js
function formatPhoneNumber(phone) {
   return phone.replace(re, '$1-$2-$3 $4$5').replace(/x$/, '');
}
```

That should be clean set up for a unit test